examples/specs/normalized/errorband_2d_horizontal_color_encoding_normalized.vl.json

```diff
@@ -41,7 +41,7 @@
               "title": "Year (year)",
               "type": "temporal",
               "field": "year_Year",
-              "axis": {"format": "%Y"}
+              "axis": {"format": "%Y", "formatType": "time"}
```

Comment:
It doesn't seem necessary to set the timeFormat when the type is already `temporal`. Can you change this?